src/core/evaluator.js

```diff
@@ -675,6 +675,7 @@ class PartialEvaluator {
           width: imgData.width,
           height: imgData.height,
           interpolate: imgData.interpolate,
+          count: 1,
```

Comment:
Won't the `QueueOptimizer` potentially break this, or at make things less efficient, unless this new parameter is *also* added to the list in:
https://github.com/mozilla/pdf.js/blob/fffce7910e63038cc15cd01522a768ee86e081be/src/core/operator_list.js#L253-L265